bundles/org.openhab.binding.solarman/src/main/java/org/openhab/binding/solarman/internal/util/StringUtils.java

```diff
@@ -0,0 +1,67 @@
+/**
+ * Copyright (c) 2010-2024 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.solarman.internal.util;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.eclipse.jdt.annotation.Nullable;
+
+/**
+ * The {@link StringUtils} common string utility functions
+ *
+ * @author Catalin Sanda - Initial contribution
+ */
+@NonNullByDefault
+public class StringUtils {
+    public static final String EMPTY = "";
+
+    public static boolean isEmpty(@Nullable CharSequence cs) {
+        return cs == null || cs.length() == 0;
+    }
+
+    public static boolean isNotEmpty(@Nullable CharSequence cs) {
+        return !isEmpty(cs);
+    }
+
+    public static String replaceChars(final String str, final String searchChars, @Nullable String replaceChars) {
+        if (isEmpty(str) || isEmpty(searchChars)) {
```

Comment:
```suggestion
        if (str.isBlank()) || searchChars.isBlank()) {
```

Prefer default java api above custom utility methods.